templates/page/page.component.inc

```diff
@@ -80,4 +80,20 @@ function europa_preprocess_page(&$variables, $hook) {
   }
   $variables['site_header'] = array('#theme' => array('site_header'));
   $variables['messages'] = $variables['show_messages'] ? array('#theme' => array('status_messages')) : '';
+  $variables['page_header'] = array(
+    '#theme' => array('page_header'),
+    '#header_bottom' => $variables['page']['header_bottom'],
+    '#site_name' =>  $variables['site_name'],
+    '#title_prefix' => $variables['title_prefix'],
+    '#title_suffix' => $variables['title_suffix'],
+    '#page_title' => drupal_get_title(),
+    '#page_intro' => NULL,
+    '#page_meta' => NULL,
+    '#meta' => array('#theme' => array('meta')),
+  );
+  if (!empty($variables['node'])) {
```

Comment:
The page introduction is also generated on line 39 (using the variable ec_europa_introduction_field). 
We shouldn't assume the the introduction will come from a unique field so we should probably figure out a method of making it dynamic (a setting in the node type for example).